src/containers/shared/components/Transaction/AccountDelete/index.ts

```diff
@@ -1,9 +1,15 @@
-import { TransactionMapping } from '../types'
+import {
+  TransactionAction,
+  TransactionCategory,
+  TransactionMapping,
+} from '../types'
 
 import { Simple } from './Simple'
 import { AccountDelete } from './types'
 
 export const AccountDeleteTransaction: TransactionMapping = {
   Simple,
   parser: (tx: AccountDelete): AccountDelete => tx,
+  action: TransactionAction.CANCEL,
```

Comment:
The name `action` feels a little confusing to me, especially given that we have redux actions-related pages still in the repo.

Reply:
Open to other names. TransactionVerb?

Reply:
Sure

Reply:
bump

Reply:
I was joking about calling it TransactionVerb, I should have made that clearer :) .  I could not come up with a better name.  I don't think it will be cause too much confusion with redux actions as they are typed.

Reply:
Best I can think of is maybe `TransactionCategory`?

Reply:
I just added `TransactionCategory`.  The tricky part is I needed to add names for two new taxonomies.

Reply:
I used `Category` and `Type` in my initial breakdown

Reply:
`Action` is fine for now, we can always change it later if we decide it's confusing

Reply:
> I used `Category` and `Type` in my initial breakdown

I was worried `Type` would get confused with `TransactionType`.